api/src/main/java/org/openmrs/module/dhisreporting/Configurations.java

```diff
@@ -19,6 +19,10 @@ private Concept getConceptByGpCode(String globalPropertyCode) {
 		return conceptId != null ? Context.getConceptService().getConcept(conceptId) : null;
 	}
 
+	public Concept getDxfToAdxSwitchConcept() {
+		return getConceptByGpCode(DHISReportingConstants.DXFTOADXSWITCH_CONCEPTID);
```

Comment:
This will instead be of Boolean dataType, it should look like; https://github.com/jembi/openmrs-module-dhisconnector/blob/master/api/src/main/java/org/openmrs/module/dhisconnector/Configurations.java#L10